asset-transfer-basic/application-gateway-java/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/6.5/userguide/tutorial_java_projects.html
+ */
+plugins {
+    // Apply the java plugin to add support for Java
+    id 'java'
+    id 'eclipse'
```

Comment:
why is the eclipse plugin needed?

Reply:
This is not needed. have removed eclipse plugin